src/OCServo.h

```diff
@@ -0,0 +1,89 @@
+#ifndef OCSERVO_H
+#define OCSERVO_H
+
+#include <Arduino.h>
+#include <SoftwareSerial.h>
+
+#include "ROBSMotor.h"
+
+enum OCSMode {
+    OCS_SERVO,
+    OCS_MOTOR
+};
+
+enum OCSResponseLevel {
+    OCS_RESPOND_READ_ONLY,
+    OCS_RESPOND_ALL
+};
+
+typedef enum e_SerialPortType {
+    SERIAL_HARDWARE,
+    SERIAL_SOFTWARE
+} SerialPortType;
+
+struct OCSResponse {
+    byte prefix[2];
+    byte id;
+    byte length;
+    byte instruction;
+    int numberOfParameters;
+    byte parameters[16];
+    byte checksum;
+};
```

Comment:
Using fixed-length types like uint16_t or uint32_t ensures consistent and predictable behavior across different platforms, as the size of standard types (int, long) can vary. This makes code more portable and helps prevent subtle bugs caused by unexpected size differences. Please fix in the rest places.

```suggestion
/**
 * @brief Structure representing a response from OCServo
 */
struct OCSResponse {
    uint8_t prefix[2];
    uint8_t id;
    uint8_t length;
    uint8_t instruction;
    uint16_t parameterCount;
    uint8_t parameters[16];
    uint8_t checksum;
};
```